packages/terra-draw-google-maps-adapter/src/terra-draw-google-maps-adapter.spec.ts

```diff
@@ -401,6 +401,107 @@ describe("TerraDrawGoogleMapsAdapter", () => {
 			expect(lngLatFromEvent?.lng).toEqual(testLng);
 			expect(lngLatFromEvent?.lat).toEqual(testLat);
 		});
+
+		it("returns uses the fullscreen element for boundingClientRect in fullscreen mode", () => {
+			// Mock fullscreen element with its own getBoundingClientRect
+			const fullscreenElementMock = {
+				getBoundingClientRect: jest.fn(() => ({
+					left: 0,
+					top: 0,
+				})),
+			};
+
+			// Store original document.fullscreenElement
+			const originalFullscreenElement = Object.getOwnPropertyDescriptor(
```

Comment:
If you've got guidance on better ways to handle this, happy to do so. The `jest.spyOn` approach you used elsewhere doesn't work for this particular approach (I tried)